test/controllers/projects/admin_requests_controller_test.rb

```diff
@@ -10,7 +10,7 @@ def test_post_admin_request
         project_id: eol_project.id,
         email: {
           subject: "Admin request subject",
-          message: "Message for admins"
+          content: "Message for admins"
         }
```

Comment:
This has been wrong for awhile (the param name in the form is definitely `content`) but somehow was passing. 

This fix was needed to get it passing now.